src/adapters/poi/idunn_poi.js

```diff
@@ -16,17 +16,21 @@ export default class IdunnPoi extends Poi {
     this.address = rawPoi.address
   }
 
-  static async poiApiLoad(id) {
+  static async poiApiLoad(id, options = {}) {
     let rawPoi = null
     let url = `${serviceConfig.idunn.url}/v1/pois/${id}`
+    let requestParams = {}
+    if(options.simple) {
+      requestParams = {verbosity : 'short'}
```

Comment:
`verbosity` param is not implemented for the old `/v1/pois` route.
Use `/v1/places` instead